h2o-r/h2o-package/R/parse.R

```diff
@@ -30,22 +30,25 @@
 #'        acquired by calling \link{h2o.decryptionSetup}.
 #' @param chunk_size size of chunk of (input) data in bytes
 #' @param skipped_columns a list of column indices to be excluded from parsing
+#' @param custom_non_data_line_markers (Optional) If a line in imported file starts with any character in given string it will NOT be imported. Empty string means all lines are imported, NULL means that default behaviour for given format will be used
```

Comment:
```suggestion
#' @param custom_non_data_line_markers (Optional) If a line in an imported file starts with any character in the given string, it will be skipped and NOT be imported. An empty string means all lines are imported, NULL means that default behavior for given format will be used.
```